create_docker_sysext.sh

```diff
@@ -3,25 +3,37 @@ set -euo pipefail
 
 ARCH="${ARCH-x86_64}"
 OS="${OS-flatcar}"
+ONLY_CONTAINERD="${ONLY_CONTAINERD-0}"
+ONLY_DOCKER="${ONLY_DOCKER-0}"
+FORMAT="${FORMAT-squashfs}"
 
 if [ $# -lt 2 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
   echo "Usage: $0 VERSION SYSEXTNAME"
   echo "The script will download the Docker release tar ball (e.g., for 20.10.13) and create a sysext squashfs image with the name SYSEXTNAME.raw in the current folder."
   echo "A temporary directory named SYSEXTNAME in the current folder will be created and deleted again."
   echo "All files in the sysext image will be owned by root."
   echo "The necessary systemd services will be created by this script, by default only docker.socket will be enabled."
+  echo "To only package containerd without Docker, pass ONLY_CONTAINERD=1 as environment variable (current value is '${ONLY_CONTAINERD}')."
+  echo "To only package Docker without containerd and runc, pass ONLY_DOCKER=1 as environment variable (current value is '${ONLY_DOCKER}')."
   echo "To use arm64 pass 'ARCH=aarch64' as environment variable (current value is '${ARCH}')."
   echo "To build for another OS than Flatcar, pass 'OS=myosid' as environment variable (current value is '${OS}'), e.g., 'fedora' as found in 'ID' under '/etc/os-release'."
   echo "The '/etc/os-release' file of your OS has to include 'SYSEXT_LEVEL=1.0' as done in Flatcar."
+  echo "If you miss the mksquashfs tool you can pass FORMAT=btrfs or FORMAT=ext4 as environment variable but the files won't be owned by root."
```

Comment:
To miss a tool sounds weird (you usually miss someone close when you don't see them for a long time).

```suggestion
  echo "If the mksquashfs tool is missing you can pass FORMAT=btrfs or FORMAT=ext4 as environment variable but the files won't be owned by root."
```